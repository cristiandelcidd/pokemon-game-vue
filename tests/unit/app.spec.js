import { shallowMount } from "@vue/test-utils";

import App from "@/App.vue";

describe("App component", () => {
  test("should match with the snapshot", () => {
    const wrapper = shallowMount(App);

    expect(wrapper.html()).toMatchSnapshot();
  });
});
