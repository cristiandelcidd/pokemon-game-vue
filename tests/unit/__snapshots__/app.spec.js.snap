// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component should match with the snapshot 1`] = `<pokemon-page-stub></pokemon-page-stub>`;
